cli/standalone/binary.rs

```diff
@@ -129,66 +133,94 @@ pub fn load_npm_vfs(root_dir_path: PathBuf) -> Result<FileBackedVfs, AnyError> {
     root_path: root_dir_path,
     start_file_offset: trailer.npm_files_pos,
   };
-  Ok(FileBackedVfs::new(file, fs_root))
+  Ok(FileBackedVfs::new(data.to_vec(), fs_root))
```

Comment:
For future: FileBackedVfs can be made a zero-copy on `data` because sui returns a slice :)